plugins/woocommerce/src/StoreApi/Schemas/V1/AbstractAddressSchema.php

```diff
@@ -130,6 +131,10 @@ function( $carry, $key ) use ( $address, $validation_util ) {
 						break;
 					default:
 						$carry[ $key ] = $address[ $key ];
+						// Specific sanitization for string types, skipping other types from being coerced to strings.
+						if ( 'string' === $field_schema[ $key ]['type'] ) {
+							$carry[ $key ] = wp_kses( $address[ $key ], [] );
```

Comment:
If this is not a string, is the value still sanitised somehow? I see the other method uses `rest_sanitize_value_from_schema`.

Reply:
True, not sure why they're different. I added `rest_sanitize_value_from_schema` to this for values we don't already handle (`country`, `state`, and `postcode`)